### ISSUE TITLE
ltrdigest: Assertion failed: (rval == 0), function gt_ltrdigest_pdom_visitor_feature_node

### ISSUE THREAD

**Issue by sestaton** [genometools/genometools#604]
Full error message:

    Assertion failed: (rval == 0), function gt_ltrdigest_pdom_visitor_feature_node, file src/ltr/ltrdigest_pdom_visitor.c, line 865.
    This is a bug, please report it at
    https://github.com/genometools/genometools/issues
    Please make sure you are running the latest release which can be found at
    http://genometools.org/pub/
    You can check your version number with `gt -version`.
    trim.sh: line 35: 23274 Aborted                 (core dumped) $gt ltrdigest -trnas $trnas -hmms $hmm -aliout yes -aaout yes -seqfile $db -matchdescstart -seqnamelen 50 -o $gff_h -outfileprefix ltrdigest_trim $gff_sort

I am using the latest unstable release (downloaded on June 1st). If I run LTRharvest with the `-similar 99` option I do not see these errors. If I lower the similarity threshold, to the default for example, then I get this error with LTRdigest. I can give more information about the input or try to create a test with a public data set if that would be helpful.

**Comment by satta** [comment]
Thanks for the report! That is indeed very strange, especially given the fact that the assertion in question is triggered when a pipe couldn't be created -- a technical error I wouldn not expect to be related to a given input. I would be very interested to take a look at the input you used and trace the problem. Can you isolate a minimal example triggering the issue?

**Comment by satta** [comment]
Any news on this one?

**Comment by sestaton** [comment]
Sorry for the delay, it took a lot of testing to solve the issue. First, I worked with Arabidopsis and couldn't recreate the problem, then I tried with maize and still I could not recreate this message. As it turns out, it was an issue with the data I am using. Specifically, one of the pseudomolecules I was given was concatenated scaffolds (with small artificial gaps) that could not be placed. This was something I would not have included anyway, so removing this one pseudomolecule solved the problem.
